src/client/components/MessageCenter/Topic/Footer/Footer.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useCallback, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useParams } from 'react-router-dom'
+
+import { Objects } from 'utils/objects'
+
+import { MessageTopic, MessageTopicType } from 'meta/messageCenter'
+
+import { useAppDispatch } from 'client/store'
+import { useAssessment, useCycle } from 'client/store/assessment'
+import { MessageCenterActions } from 'client/store/ui/messageCenter'
+import { useCountryIso } from 'client/hooks'
+import { DataCell, DataGrid } from 'client/components/DataGrid'
+import TextArea from 'client/components/Inputs/TextArea'
+
+import { useShowActions } from './hooks/useShowActions'
+
+type Props = { topic: MessageTopic }
+const Footer: React.FC<Props> = (props: Props) => {
+  const { topic } = props
+  const { t } = useTranslation()
+  const [message, setMessage] = useState('')
+  const dispatch = useAppDispatch()
+  const countryIso = useCountryIso()
+  const assessment = useAssessment()
+  const cycle = useCycle()
+  const { canResolve, canPostMessage } = useShowActions(topic)
```

Comment:
as afore mentioned, this is extracted from Topic.tsx and this hook is introduced, otherwise it should be 1:1